docs/audit-log-filter/audit-log-filter-options.md

```diff
@@ -0,0 +1,419 @@
+# Audit Log Filter options, variables, and functions
+
+## Options
+
+If you specify a value that is incorrect at server startup, the plugin may fail and not be loaded by the server. If the plugin fails to load, the server may generate error messages for other audit log settings.
+
+The following list are the available variables for the Audit Log filter feature:
+
+audit_log_filter_handler
+audit_log_filter_syslog_ident
+audit_log_filter_syslog_facility
+audit_log_filter_syslog_priority
+
+### --audit-log-filter
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Enumeration  |
+| Default | ON  |
+
+Defines how the server loads the plugin. You must have registered or loaded the plugin using either `INSTALL_PLUGIN` or `--plugin-load` or `--plugin-load-add`. 
+
+The value can only be a plugin-loading operation. Enabling the plugin exposes other options and status variables that control logging.
+
+The valid values are the following:
+
+| Values |
+| --- |
+| ON |
+| OFF |
+| FORCE |
+| FORCE_PLUS_PERMANENT |
+
+### audit_log_filter_buffer_size
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-buffer-size |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 1048576  |
+| Minimum value | 4096 |
+| Maximum value | 18446744073709547520 |
+| Units | byes |
+| Block size | 4096 |
+
+Defines the buffer size in multiples of 4096 when logging is asynchronous. The contents for events are stored in a buffer. The contents are stored until the contents are written.
+
+The plugin initializes a single buffer and removes the buffer when the plugin terminates.
+
+### audit_log_filter_compression
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-compression |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Enumeration  |
+| Default | NONE  |
+| Valid values | NONE or GZIP |
+
+Defines the compression type for the audit log filter file. The values can be either `NONE`, the default value and no compression) or `GZIP`.
+
+
+### audit_log_filter_database
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-database |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | String  |
+| Default | mysql  |
+
+Defines the audit_log_filter database, used to store the necessary tables, and is read-only. Set this option at system startup. The database name cannot exceed 64 characters or be `NULL`.
+
+Using an invalid database name prevents the use of audit log filter tables.
```

Comment:
prevents the use of audit log filter tables -> prevents the use of audit log filter plugin ?

Reply:
updated.